docs/book/src/developer/tilt.md

```diff
@@ -68,8 +87,12 @@ enable_providers:
 kustomize_substitutions:
   IBMCLOUD_API_KEY: "XXXXXXXXXXXXXXXXXX"
   POWERVS_PROVIDER_ID_FORMAT: "v2"
+  EXP_CLUSTER_RESOURCE_SET: "true"
+  CLUSTER_TOPOLOGY: "true"
```

Comment:
where these 2 variables are referenced? I don't see them anywhere in the code?

Reply:
Any supporting document who is using these variables will be great.

Reply:
ClusterClass is a Experimental feature: https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-class/index.html.
Enabling the feature gate: https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/book/src/user/quick-start.md#enabling-feature-gates

Reply:
do you mind adding same information into the doc?

Reply:
Yes sure, I can do that